config.go

```diff
@@ -95,6 +96,7 @@ func defaultConfig() sequinsConfig {
 		S3: s3Config{
 			Region:          "",
 			AccessKeyId:     "",
+			MaxRetries:      2,
```

Comment:
Small nit, I thing retrying 3 times is better, mostly for consistency with other retries in sequins.